docs/topics/api/collections.rst

```diff
@@ -18,9 +18,9 @@ List
 This endpoint allows you to list all collections authored by the specified user.
 The results are sorted by the most recently updated collection first.
 
-
 .. http:get:: /api/v3/accounts/account/(int:user_id|string:username)/collections/
 
+    :query int has_addon: Optional add-on id parameter allowing you to check if a given add-on is part of the collections being returned. When passed, each collection object in the results will have an extra `has_addon` boolean property indicating whether the add-on is present in this collection or not.
```

Comment:
improvement idea: have `has_addon` support the styles of add-on ID (i.e. pk and slug) that Collection Add-ons Create API endpoint supports.

Reply:
This is hard to do without forcing an extra join. Currently `with_has_addon()` only looks in the m2m table, so it only has the id.

Reply:
okay, fair enough.